src/App.js

```diff
@@ -21,31 +21,50 @@ function useApp() {
   return { centralTeams, pacificTeams };
 }
 
+function SortTeamsByRank(teams, year) {
+  const teamsInTheYear = teams.map((team) => {
+    const result = team.results.find((el) => el.year === year);
+    return {
+      id: team.id,
+      name: team.name,
+      founded_year: team.founded_year,
+      league: team.league,
+      result: result,
+    };
+  });
+
+  return teamsInTheYear.sort(function (a, b) {
+    return a.result.rank - b.result.rank;
+  });
```

Comment:
Why not using arrow func? I guess you don't have preference with it :)

```suggestion
  return teamsInTheYear.sort((a, b) => (a.result.rank - b.result.rank));
```

Reply:
おっしゃる通りですね。。適用しました！